docs/Other SDL Features/Remote Control Vehicle Features/index.md

```diff
@@ -188,33 +196,222 @@ if (sdlManager.getSystemCapabilityManager().isCapabilitySupported(SystemCapabili
 ```
 !@
 
-### Getting Data
-Once you know you have permission to use the remote control feature, you can retrieve the data. The following code is an example of how to get data from the radio module. The example also subscribes to updates to radio data, which will be discussed later on in this guide.
+### Setting The User's Seat
+
+!!! NOTE
+This section only applies when connected to SDL 6.0+ head units
+!!!
+
+The first step before attempting to set any module is to have the user select their seat. Seat location may affect which modules the user is permitted to control depending on the OEMs rules. The default seat location is `Driver`. Seat location can be updated by setting the `userLocation` property in the `SDLSetGlobalProperties` RPC and sending it.
+
+In a real-life scenario, you may wish to show the user a map or list of all available seats in order to ask them where they are located. This example is only meant to show you how to access the available data, and not how to build your UI/UX. 
+
+An array of seats can be found in the `seatLocationCapability`s `seat` array. Each `SeatLocation` object within the `seats` array will have a `grid` struct. This struct will tell you the seat placement of that particular seat. This information can be very useful for creating a map or list for users to select from.
+
+The `grid` system starts with the top left seat being (0,0,0).  For example, assuming a United States vehicle, a `grid` of `col`=0, `row`=0 and `level`=0 would be the drivers' seat. A `col`=2, `row`=0 and `level`=0 would be referring to the front right passenger location, assuming the car has 3 columns. A negative `col` or `row` means it is outside the vehicle. The `colspan` and `rowspan` properties tell you how many rows and columns that module or seat takes up.
+
+![Car](assets/Car.png)
+
+| col=-1, row=-1  | col=0 | col=1 | col=2 |
+| --- | --- | --- | --- |
+| row=0 | driver's seat: {col=0, row=0, level=0, colspan=1, rowspan=1, levelspan=1} |   | front passenger's seat : {col=2, row=0, level=0, colspan=1, rowspan=1, levelspan=1}|
+| row=1 | rear-left seat : {col=0, row=1, level=0, colspan=1, rowspan=1, levelspan=1}| rear-middle seat :  {col=1, row=1, level=0, colspan=1, rowspan=1, levelspan=1} | rear-right seat : {col=2, row=1, level=0, colspan=1, rowspan=1, levelspan=1} |
+
+@![iOS]
+##### Objective-C
+```objc
+[self.sdlManager.systemCapabilityManager subscribeToCapabilityType:SDLSystemCapabilityTypeSeatLocation withBlock:^(SDLSystemCapability * _Nonnull capability) {
+    if (!capability) {
+        return;
+    }
+    NSArray<SDLSeatLocation *> *seats = capability.seatLocationCapability.seats;
+    <#Save Remote Capabilities#>
+}];
+```
+
+##### Swift
+```swift
+sdlManager.systemCapabilityManager.subscribe(toCapabilityType: .seatLocation, with: { capability in
+    guard let seats = capability.seatLocationCapability?.seats else { return }
+    <#Save Remote Capabilities#>
+})
+```
+!@
```

Comment:
TODOs for Java code are missing basically everywhere.

Reply:
added